src/main/webapp/app/core/about-us/about-us.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
+import { filter, tap } from 'rxjs/operators';
+import { ProfileInfo } from 'app/shared/layouts/profiles/profile-info.model';
+import { SERVER_API_URL } from 'app/app.constants';
+import { StaticContentService } from 'app/shared/service/static-content.service';
+import { AboutUsModel } from 'app/core/about-us/models/about-us-model';
+
+@Component({
+    selector: 'jhi-about-us',
+    templateUrl: './about-us.component.html',
+    styles: [],
+})
+export class AboutUsComponent implements OnInit {
+    private readonly issueBaseUrl = 'https://github.com/ls1intum/Artemis/issues/new?projects=ls1intum/1';
+    readonly bugReportUrl = `${this.issueBaseUrl}&labels=bug&template=bug-report.md`;
+    readonly featureRequestUrl = `${this.issueBaseUrl}&labels=feature&template=feature-request.md`;
+    email: string;
+
+    data: AboutUsModel;
+
+    readonly SERVER_API_URL = SERVER_API_URL;
+
+    constructor(private route: ActivatedRoute, private profileService: ProfileService, private staticContentService: StaticContentService) {}
+
+    /**
+     * On init get the json file from the Artemis server and save it.
+     * On init get the mail data needed for the contact
+     */
+    ngOnInit(): void {
+        this.staticContentService.getStaticJsonFromArtemisServer('/about-us.json').subscribe((data) => {
```

Comment:
So it looks like `private readonly staticContentUrl = SERVER_API_URL + 'public/content/';` therefore this will create `public/content//about-us.json` 
```suggestion
        this.staticContentService.getStaticJsonFromArtemisServer('about-us.json').subscribe((data) => {
```

Reply:
Fixed it